src/frontend/shortlist/src/views/CategorizeView.vue

```diff
@@ -196,12 +199,25 @@ export default {
       this.myShortlists[e.listId].settings = JSON.parse(
         JSON.stringify(e.settings)
       );
-      // TODO: send update to api
-      console.log("SEND UPDATE SETTINGS:", e.settings);
+      // send update to api
+      // console.log("SEND UPDATE SETTINGS:", e.settings);
+      this.saveList(e.listId);
     },
     shareList(e) {
       console.log("SHARED LIST #:", e);
     },
+    deleteFromList() {
+      // console.log("DELETE FROM LIST: ", this.schoolDetailModalData.data.schoolMetadata.id);
+      let schoolIdx = this.schoolDetailModalData.schoolIdx;
+      let listIdx = this.schoolDetailModalData.listIdx;
+      this.myShortlists[listIdx].school_ids.splice(schoolIdx, 1);
```

Comment:
I don't remember myShortlists[listIdx] was. Is this line supposed to mark the school as neither trashed or accepted?

Reply:
This just removes the school from the list, separate from marking it trashed or accepted, but I will add a call to mark as rejected